modules/claims_api/spec/requests/v2/power_of_attorney_requests/index/rswag_spec.rb

```diff
@@ -44,6 +44,7 @@
           ],
           'decision' => {
             'statuses' => %w[
+              None
```

Comment:
This might be a swagger ui limitation, but is there way to get the query schema to show up as a schema? That way users can fully know the status enum?

An endpoint with an example + schema tabs: 
![image](https://github.com/department-of-veterans-affairs/vets-api/assets/144135615/13103b03-c7cb-421c-8617-a508e37e0a92)


This endpoint: 
![image](https://github.com/department-of-veterans-affairs/vets-api/assets/144135615/2c328631-c19d-42a4-bacb-890479df5d17)

Reply:
Yeah it's a limitation of Swagger UI [apparently](https://github.com/swagger-api/swagger-ui/issues/7696).

Reply:
So I could be misinterpreting this, but it seems like the [lighthouse standards](https://department-of-veterans-affairs.github.io/va-api-standards/paths-and-operations/custom-operations/#example-query-request) call for using the request body rather than query params. Probably for consistency in the docs since Swagger UI has the limitation cited above.

Reply:
I believe that LH documentation shows that example as a way of calling out a _custom_ search resource where the query involves PII.
> In this example, which contains (mock) PII, we're sending a prescription search query that includes a mock SSN number in a POST body.

[Elsewhere](https://department-of-veterans-affairs.github.io/va-api-standards/paths-and-operations/paths/#collection-resources) they show a search resource that uses query parameters.
> GET ../rx/v1/prescriptions?page=2&pageSize=10

I suspect that in particular they _did not_ have this nuanced Swagger UI limitation in mind during the development of their standards.

Reply:
@stiehlrod We also did have this discussion in an [earlier PR](https://github.com/department-of-veterans-affairs/vets-api/pull/16661#discussion_r1599118403) and then went on to capture that thinking in a backlogged [discovery ticket](https://jira.devops.va.gov/browse/API-35332) to revisit in the future.